packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -1276,6 +1281,21 @@ describe('plugin-meetings', () => {
           assert.equal(webex.meetings.preferredWebexSite, 'go.webex.com');
         });
 
+        const setup = ({me = {type: 'validuser'}} = {}) => {
+          Object.assign(webex.people,{
+            _getMe: sinon.stub().returns(Promise.resolve(me)),
+          })
+        }
+
+        it('should not call request.getMeetingPreferences if user is a guest', async () => {
+          setup({me: {type: 'appuser'}});
```

Comment:
Do we need a separate setup function if it is not being used elsewhere?

Reply:
> Do we need a separate setup function if it is not being used elsewhere?

Done removed the setup function @mkesavan13